village/src/App.css

```diff
@@ -1,3 +1,25 @@
 .App {
   text-align: center;
+  background-image: url('http://hdqwalls.com/wallpapers/smurfs-the-lost-village-4k-5k-ap.jpg');
+  background-size: 100%;
+  object-fit: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  font-family: 'Luckiest Guy', cursive;
+  letter-spacing: 2px;
+  background-attachment: fixed;
+}
+
+.overlay {
+  background: linear-gradient(
+    180deg,
+    rgba(255, 255, 255, 0) 49%,
+    rgba(0, 27, 108, 0.9528186274509804) 100%
+  );
+  position: absolute;
```

Comment:
Not needed, it gets overwritten to `position: fixed` anyway but just in terms of cleaning things up.  Assuming that you want it to be fixed so as more are added they don't exceed the container and then appear on a white background versus the background of the container originally.